django-stubs/contrib/contenttypes/forms.pyi

```diff
@@ -1,8 +1,12 @@
-from typing import Any, Optional
+from typing import Any, Optional, Type, TypeVar
 
-from django.forms.models import BaseModelFormSet
+from django.db.models import Model
+from django.forms.models import BaseModelFormSet, ModelForm
 
-class BaseGenericInlineFormSet(BaseModelFormSet):
+_M = TypeVar("_M", bound=Model)
+_ModelFormT = TypeVar("_ModelFormT", bound=ModelForm)
+
+class BaseGenericInlineFormSet(BaseModelFormSet[_M, _ModelFormT]):
```

Comment:
Should we add it to our monkeypatching? Since it is generic now.

Reply:
`BaseModelFormSet` doesn't seem to be patched either.